chain-abci/src/app/commit.rs

```diff
@@ -168,12 +168,13 @@ impl<T: EnclaveProxy> ChainNodeApp<T> {
             }
         }
 
-        inittx.put(
-            COL_APP_STATES,
-            &i64::encode_var_vec(new_state.last_block_height),
-            &new_state.last_apphash,
-        );
-        inittx.put(COL_NODE_INFO, LAST_STATE_KEY, &new_state.encode());
+        let encoded_state = new_state.encode();
+        inittx.put(COL_NODE_INFO, LAST_STATE_KEY, &encoded_state);
+        let encoded_height = i64::encode_var_vec(new_state.last_block_height);
+        inittx.put(COL_APP_HASHS, &encoded_height, &new_state.last_apphash);
```

Comment:
COL_APP_HASH?
because it's single

Reply:
It's actually a map of `height -> app_hash`